server/controllers/updateIsSitter.js

```diff
@@ -0,0 +1,33 @@
+const Profile = require("../models/Profile");
+
+const getSitterStatus = async (_id) => {
+  const response = await Profile.find({ _id }, "isSitter", (err, result) => {
+    if (err) throw new Error(err);
+    return result;
+  });
+  return response;
+};
+
+const changeIsSitter = async (_id, value, res) => {
```

Comment:
Question: Don't we have a route to update a user's profile, maybe consider using that instead of creating a new route :)

Reply:
I fixed the editProfile function in the profile controller, and now fetch uses a post method to send data to the profile controller. So this controller was deleted in favor of the dynamic profile controller.